File: source/api/recipients/create/_create.cs
var recipientInfo = new RecipientCreateInfo();
recipientInfo.Name = "Somchai Prasert";
recipientInfo.Email = "somchai.prasert@example.com";
recipientInfo.RecipientType = RecipientType.Individual;
recipientInfo.BankAccount = new BankAccountInfo()
{
  Brand = "bbl",
  Number = "1234567890",
  Name = "SOMCHAI PRASERT"
};

var recipient = client.RecipientService.CreateRecipient(recipientInfo);

File: source/api/recipients/create/_create.php
<?php

$recipient = OmiseRecipient::create(array(
  'name' => 'Omise Tester',
  'description' => 'Tester account',
  'email' => 'tester@omise.co',
  'type' => 'individual',
  'tax_id' => '',
  'bank_account' => array(
    'brand' => 'bbl',
    'number' => '1234567890',
    'name' => 'Tester Account'
  )
));
